.github/workflows/ci.yml

```diff
@@ -488,6 +488,111 @@ jobs:
       - run: python3 -m pip install --upgrade pip && pip install nox
       - run: python3 -m nox -s check-feature-powerset
 
+  test-cross-compilation:
+    needs: [fmt]
+    # if: ${{ contains(github.event.pull_request.labels.*.name, 'CI-build-full') || (github.event_name != 'pull_request' }}
```

Comment:
This for testing this without attaching `CI-build-full`, right?

Reply:
Yep exactly, I didn't want to burn through CPUs while testing this out 😂